src/components/FilmForm/FilmForm.tsx

```diff
@@ -101,7 +138,9 @@ const FilmForm: React.FC<FormProps> = ({ film, modalType }) => {
         value={formik.values.poster_path}
       />
       <label htmlFor={Formfields.genres}>GENRE</label>
-
+      {formik.errors.genres ? (
+        <div className="error">{formik.errors.genres}</div>
+      ) : null}
```

Comment:
too many duplication, could be simplified to one row component error or moved to custom input